cluster/src/main/java/com/hartwig/pipeline/execution/vm/VirtualMachineJobDefinition.java

```diff
@@ -129,6 +129,14 @@ static VirtualMachineJobDefinition amber(BashStartupScript startupScript, Result
                 .build();
     }
 
+    static VirtualMachineJobDefinition virusbreakend(BashStartupScript startupScript, ResultsDirectory resultsDirectory) {
+        return ImmutableVirtualMachineJobDefinition.builder()
```

Comment:
this will use the default machine size of 32 cpu 120G mem. Do we have any idea of the performance profile? GRIDSS uses 12/64.

Reply:
No idea to be honest but I can't imagine VirusBreakend using more resources than GRIDSS, so I've now configured 12/64 as well.

Reply:
Discussed and will configure to 12 cores, 40 GB mem. Should cover it.